packages/react/cypress/component/auto/table/AutoTableSort.cy.tsx

```diff
@@ -72,31 +72,31 @@ describe("AutoTable - Sort", () => {
     cy.wait("@getWidgets").its("request.body.variables").should("deep.equal", { first: 50 });
 
     mockGetWidgetsWithDescendingSort();
-    cy.contains("Sortable Id").click({ force: true });
+    cy.contains("ID").click({ force: true });
```

Comment:
Now that we use the default selection map from the real API client, the fields have to be available in the real model. So I have to rename these two fields.